src/styles/stopwatch__minute-hand.scss

```diff
@@ -0,0 +1,21 @@
+@import 'variables';
+
+.stopwatch__minute-hand {
+  position: absolute;
+  width: $minute-hand-width;
+  height: $minute-hand-length;
+  background-color: $minute-hand-color;
+  top: calc(50% - #{$minute-hand-length});
+  left: 48.5%;
+  transform-origin: center bottom;
+  animation: rotateMinute 3600s steps(60) infinite;
```

Comment:
The animation property references `rotateMinute`, but the keyframes are defined as `rotate-minute`. Ensure that the keyframe name matches the animation name to avoid any runtime errors.